src/pattern-library/components/patterns/DialogComponents.js

```diff
@@ -188,9 +189,9 @@ function LongModalExample() {
         onCancel={() => setDialogOpen(false)}
         title="Modal with overflowing content"
       >
-        <div className="hyp-scrollbox">
+        <Scrollbox classes="px-4 py-2">
```

Comment:
Part of the move toward removing the patterns layer, but sort of distracting here, admittedly.

Reply:
My goal in this pass is remove dependencies in the Foundations and Components patterns on any Patterns, uh, patterns. It's arguable both ways whether to do this now or not. But it doesn't hurt. Patterns are going away very soon.